src/main/java/com/ericsson/gerrit/plugins/eiffel/events/models/EiffelSourceChangeSubmittedEventEventParams.java

```diff
@@ -0,0 +1,32 @@
+/*
```

Comment:
Remove one "Event" in this class name or change the "EiffelSourceChangeCreatedEventParams.java" to "EiffelSourceChangeCreatedEventEventParams.java", I am fine with either.

Reply:
Done.

Reply:
Done